lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -10175,22 +10175,8 @@ namespace Js
 
     BOOL JavascriptOperators::IsIterable(RecyclableObject* instance, ScriptContext* scriptContext)
     {
-        if (JavascriptProxy::Is(instance))
-        {
-            Var func = JavascriptOperators::GetProperty(instance, PropertyIds::_symbolIterator, scriptContext);
-            if (JavascriptOperators::IsUndefinedObject(func))
-            {
-                return FALSE;
-            }
-            else
-            {
-                return TRUE;
-            }
-        }
-        else
-        {
-            return JavascriptOperators::HasProperty(instance, PropertyIds::_symbolIterator);
-        }
+        Var func = JavascriptOperators::GetProperty(instance, PropertyIds::_symbolIterator, scriptContext);
```

Comment:
This is not allowed. See #584.

I want to remove IsIterable().  It leads to code that does not match the spec.

@nicolo-ribaudo can you find a different way.  Possibly by fixing #584?